packages/components/src/utils/colors-values.js

```diff
@@ -1,12 +1,38 @@
+/**
+ * External dependencies
+ */
+import { merge } from 'lodash';
+
 /**
  * Internal dependencies
  */
 import { rgba } from './colors';
+
 export const BASE = {
 	black: '#000',
 	white: '#fff',
 };
 
+export const G2 = {
```

Comment:
Maybe add TODO-type comment explaining what G2 means (succinctly, focused on why this object is separate from the other colors) and with a note on when it would be a good time to consolidate all of this. Otherwise other devs in a few months won't have a clue of what this means. :)